ansible/roles/streaming-whisper/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: Install dependencies
+  ansible.builtin.package:
```

Comment:
It's fine to include these, but just FYI they should both be already installed in the base image so this will be a no-op.